2023-07-16 13:27:33,034 (38474/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-07-16 13:27:33,034 (38474/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-07-16 13:27:33,036 (38474/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/main.py", line 640, in connect\n    raise e\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/main.py", line 627, in connect\n    pgexecute = PGExecute(database, user, passwd, host, port, dsn, **kwargs)\n                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 161, in __init__\n    self.connect(database, user, password, host, port, dsn, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 207, in connect\n    conn = psycopg.connect(conn_info)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/psycopg/connection.py", line 727, in connect\n    raise ex.with_traceback(None)\npsycopg.OperationalError: connection is bad: No such file or directory\n\tIs the server running locally and accepting connections on that socket?\n'
2023-07-16 13:52:12,248 (64347/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-07-16 13:52:12,248 (64347/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-07-16 13:52:43,725 (64347/MainThread) pgcli.pgexecute ERROR - sql: '?', error: SyntaxError('syntax error at or near "?"\nLINE 1: ?\n        ^')
2023-07-16 13:52:43,726 (64347/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 373, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 423, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/pgcli/pgexecute.py", line 69, in execute\n    super().execute(*args, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/3.5.0_1/libexec/lib/python3.11/site-packages/psycopg/cursor.py", line 725, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.SyntaxError: syntax error at or near "?"\nLINE 1: ?\n        ^\n'
2023-07-16 13:52:56,226 (65267/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-07-16 13:52:56,227 (65267/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-07-16 13:53:20,785 (65889/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-07-16 13:53:20,786 (65889/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-07-16 13:56:33,651 (69564/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-07-16 13:56:33,652 (69564/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2023-07-16 13:57:31,506 (70860/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2023-07-16 13:57:31,506 (70860/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2025-01-12 12:47:13,344 (76790/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2025-01-12 12:47:13,345 (76790/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2025-01-12 12:47:19,101 (76790/MainThread) pgcli.pgexecute ERROR - sql: '?', error: SyntaxError('syntax error at or near "?"\nLINE 1: ?\n        ^')
2025-01-12 12:47:19,104 (76790/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/opt/homebrew/Cellar/pgcli/4.1.0_1/libexec/lib/python3.12/site-packages/pgcli/pgexecute.py", line 406, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File "/opt/homebrew/Cellar/pgcli/4.1.0_1/libexec/lib/python3.12/site-packages/pgcli/pgexecute.py", line 460, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/opt/homebrew/Cellar/pgcli/4.1.0_1/libexec/lib/python3.12/site-packages/pgcli/pgexecute.py", line 91, in execute\n    super().execute(*args, **kwargs)\n  File "/opt/homebrew/Cellar/pgcli/4.1.0_1/libexec/lib/python3.12/site-packages/psycopg/cursor.py", line 97, in execute\n    raise ex.with_traceback(None)\npsycopg.errors.SyntaxError: syntax error at or near "?"\nLINE 1: ?\n        ^\n'
